scanpointgenerator/core/generator.py

```diff
@@ -22,14 +22,6 @@ class Generator(object):
     _generator_lookup = {}
     axes = []
 
-    def iterator(self):
```

Comment:
It's analogous, but the data it produces isn't the same. In extreme cases, it's also much more time-consuming.

The data from `produce_points` can be accessed directly, the data from `prepare` can only really be accessed via `get_point` or `iterator`.